### ISSUE TITLE
The status does not show in programs

### ISSUE THREAD

**Issue by mkohanim** [therealmysteryman/udi-sense-monitoring-polyglot#7]
Please do let me know if there's anything I can do to help with this issue. If you don't have a Polisy, please do let me know.

**Comment by therealmysteryman** [comment]
Sure I can have a look, not sure what is missing preventing the node to appear in the status of program. Do you have a sample of the code that is required to support this ?

**Comment by mkohanim** [comment]
Thank you! This is what I got from Chris:
Have him make sure the editor for the status entry is correct, an invalid editor is the most common reason to not see your status value for the node.

With kind regards,

******************************
  Michel Kohanim
  CEO

  (p) 818.631.0333
  http://www.universal-devices.com<http://www.universal-devices.com/>
******************************

**Comment by therealmysteryman** [comment]
Thanks, 

I spend sometime trying to debug the issue, I have modified the profile, and ran it throught check_node_values.py script and it pass the test no problem. I have compared this nodeserver with other that I have written I don't see why this one won't show in program status. Unless the editorID or variable I'm using is not valid as a status ?

I think the next step would be if I could see any error in ISY or debug when it loading that list to see if it throw an error.
